plugins/shared-react/src/types/pipeline/taskRun.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  V1ConfigMap,
+  V1ObjectMeta,
+  V1PersistentVolumeClaimTemplate,
+  V1Secret,
+} from '@kubernetes/client-node';
+import { RunStatus } from '@patternfly/react-topology';
+
+import { ComputedStatus, TerminatedReasons } from './computedStatus';
+import {
+  Condition,
+  PipelineTaskParam,
+  PipelineTaskRef,
+  TektonResource,
+  TektonResultsRun,
+  TektonTaskSpec,
+} from './pipeline';
+import { PLRTaskRunStep } from './pipelineRun';
+
+export type StepStatus = {
+  duration: string | null | undefined;
+  name: string;
+  status: RunStatus;
+};
```

Comment:
this seems related to PLR viz, in types let's just keep to required tkn resources